cmd/generate_repo_config/generate_repo_config.go

```diff
@@ -17,7 +17,9 @@ limitations under the License.
 // WORKSPACE and generates a stripped version of the file. The generated
 // config file should contain only the information relevant to gazelle for
 // dependency resolution, so go_repository rules with importpath
-// and name defined, plus any directives.
+// and name defined, plus any other files that load repsitories to this
```

Comment:
NIT/wording: The macro files aren't included in the generated config file, they are printed to stdout by this command to be read by the `go_repository_config` rule.